### ISSUE TITLE
redesign the small_text option / use-case

### ISSUE THREAD

**Issue by hamilton** [mozilla/metrics-graphics#511]
While displaying small multiples, I get the sense that the automatic text-sizing on smaller graphs doesn't work particularly well. We should perhaps reconsider the size for accessibility's sake.

<img width="1144" alt="screen shot 2015-10-15 at 6 20 26 pm" src="https://cloud.githubusercontent.com/assets/95735/10531665/773ef6a8-7369-11e5-9c30-621d67afd932.png">
